applications/MeshingApplication/custom_processes/metrics_hessian_process.cpp

```diff
@@ -109,32 +102,23 @@ void ComputeHessianSolMetricProcess<TDim, TVarType>::Execute()
         it_node->SetValue(ANISOTROPIC_RATIO, ratio); 
         
         // We compute the metric
-    #ifdef KRATOS_DEBUG 
-        if( it_node->Has(MMG_METRIC) == false) 
-        {
-            KRATOS_ERROR <<  " MMG_METRIC not defined for node " << it_node->Id();
-        }
-    #endif     
+#ifdef KRATOS_DEBUG 
+        KRATOS_ERROR_IF_NOT(it_node->Has(MMG_METRIC)) <<  "ERROR:: MMG_METRIC not defined for node " << it_node->Id();
+#endif       
     
         Vector& metric = it_node->GetValue(MMG_METRIC);
         
-    #ifdef KRATOS_DEBUG 
-        if(metric.size() != TDim * 3 - 3) 
-        {
-            KRATOS_ERROR << "Wrong size of vector MMG_METRIC found for node " << it_node->Id() << " size is " << metric.size() << " expected size was " << TDim * 3 - 3;
-        }
-    #endif
+#ifdef KRATOS_DEBUG 
```

Comment:
Why you always check some variables and other variable are only checked in debug mode?